Gemfile

```diff
@@ -12,6 +12,9 @@ group :documentation do
   gem 'redcarpet',     '2.3.0'
   gem 'github-markup', '1.0.0'
   gem 'relish'
+  if RUBY_VERSION '< 2.0.0'
```

Comment:
This seems to have broken things worse.  Every entry in the build matrix is failing during the `bundle install` phase with:

```
Undefined local variable or method `RUBY_VERSION' for Gemfile
```